README.md

```diff
@@ -173,6 +176,29 @@ But remember you can always override the default by using either `--public` or `
 
 Ask away. You can find me [@tjluoma](http://twitter.com/tjluoma) on Twitter, or create an issue on [Bitbucket](https://bitbucket.org/tjluoma/new-dual-repo/issues/new) or [GitHub](https://github.com/tjluoma/new-dual-repo/issues/new).
 
+## Advanced Setup
+
+The following is recommended for advanced users. If something looks too complicated or you don't understand it, maybe
+stick with the [Basic Setup](#basic-setup) above.
+
+Copy config/settings.example to config/settings and modify that file with your individual values. This leaves the new-dual-repo.sh file unchanged and puts all your settings in an untracked file.
+
```

Comment:
It bothered me that a file was needed in ~/.config/ and that the file was named github-and-bitbucket.txt. I get that it had generic credentials that could be shared across projects, but to my way of thinking, project settings belong with the project. That said - it's *your* project, so if you like your way, by all means keep your way.